nextflow.config

```diff
@@ -84,8 +84,18 @@ Optional input:
     * disable_common_germline_filter: disable the use of GnomAD to filter out common variants in the population
     from the somatic calls. The GnomAD resource is still required though as this common SNPs are used elsewhere to
     calculate the contamination (default: false)
+    * args_filter: optional arguments to the FilterMutectCalls function of GATK. default: "--min-allele-fraction 0.05 --min-reads-per-strand 1 --unique-alt-read-count 4" 
+    * funcotator: run gatk funcotator (default: false)
+    * db_funcotator: path to the database funcotator should use (can be downloaded from GATK FTP server)
```

Comment:
Parameters `--funcotator` and `--db_functotator` are redundant. You don't need to pass `--db_funcotator` alone, and if you pass `--funcotator` alone it asks you to pass the other one. Thus having a single parameter `--funcotator` that expects the path to the database should be enough.